.gitignore

```diff
@@ -5,3 +5,4 @@
 /osv-scanner
 /coverage.out
 /coverage.html
+fixtures/
```

Comment:
@G-Rath Do you remember why this line was added?

Reply:
I think this ignores all fixtures/ folders, even the ones we have in cmd/osv-scanner/fixtures for example.

Reply:
Yes, because the fixtures are crafted from different sources whose formatting might be different from what Prettier wants and (more importantly) that includes files that are deliberately not parsable i.e. invalid JSON files which are for testing behaviour in that scenario.

Reply:
but this PR also adds `.prettierignore` which has `fixtures/` folders listed?

Reply:
This was a mistake - I've opened https://github.com/google/osv-scanner/pull/825 to remove it